client/src/Components/Feelings/feelingsData.js

```diff
@@ -3,23 +3,39 @@ const feelings = [{
     'associatedFeelings': [
       'Happy',
       'Content',
-      'Optimistic'
+      'Optimistic',
+      'Powerful',
+      'Confident',
+      'Inquistive',
+      'Hopeful',
+      'Inspired',
+      'Playful',
+      'Proud',
+      'Thankful',
+      'Valued',
+      'Free',
+      'Centered'
```

Comment:
General comment to say THANK YOU for your work on this! Huge improvement to the UX and not small feat to add all these values on your part ❤️

Reply:
Love the positive feelings